data/platforms.yaml

```diff
@@ -0,0 +1,16 @@
+- os: darwin
+  proc: amd64
+- os: linux
+  proc: amd64
+- os: linux
+  proc: 386
+- os: linux
+  proc: arm6
```

Comment:
The links generated by linux 386, and windows x86 didn't seem to work, but that could be the PR.